src/test/scala/org/nlogo/models/SpellCheckTests.scala

```diff
@@ -27,22 +30,31 @@ class SpellCheckTests extends TestModels with BeforeAndAfterAll {
       throw new Exception("aspell not installed!")
 
   val dictPath = new File(getClass.getResource("/modelwords.txt").toURI).getPath
-  val aspell: ProcessBuilder = Seq(
-    "aspell",
-    "--encoding=UTF-8",
-    "--lang=en_US",
-    "--mode=html",
-    "--ignore-case",
-    "--personal", dictPath,
-    "list")
+  def aspell(inPath: Path): (JProcessBuilder, Path) = {
+    val tmpPath = Files.createTempFile("out", ".txt")
+    (new JProcessBuilder("aspell",
```

Comment:
> I always figured that those couple of words that made the tests fail locally (but not on Travis) was some kind of weird dictionary thing.

I thought the same. The dictionary _does_ vary with the Aspell version that's install on the machine running the tests.